_test/test_mpi_wrappers/test_exchange_CppMPI.m

```diff
@@ -19,47 +19,47 @@
         function test_OutOfRange(obj)
             % Test communications in test mode
             if obj.ignore_test
-                return
+                skipTest();
```

Comment:
Quite a few of these tests are skipped on whether `MPI_Test_Common.ignore_tests`  is true. Might it be worth adding a `.skip_reason` attribute to `MPI_Test_Common` that records why the test was skipped?